src/test/java/com/gradle/enterprise/bamboo/config/JsonConfigurationConverterTest.java

```diff
@@ -0,0 +1,55 @@
+package com.gradle.enterprise.bamboo.config;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class JsonConfigurationConverterTest {
```

Comment:
I don't think this test is necessary as it just testing `Jackson`

Reply:
I will keep it though since it's testing the contract (handling nullability and unknown fields specially)